src/Servers/Connections.Abstractions/ref/Microsoft.AspNetCore.Connections.Abstractions.netcoreapp.cs

```diff
@@ -100,8 +100,10 @@ public partial class DefaultConnectionContext : Microsoft.AspNetCore.Connections
     public partial class FileHandleEndPoint : System.Net.EndPoint
     {
         public FileHandleEndPoint(ulong fileHandle, Microsoft.AspNetCore.Connections.FileHandleType fileHandleType) { }
+        public FileHandleEndPoint(ulong fileHandle, Microsoft.AspNetCore.Connections.FileHandleType fileHandleType, bool ownsHandle) { }
         public ulong FileHandle { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
         public Microsoft.AspNetCore.Connections.FileHandleType FileHandleType { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
+        public bool OwnsHandle { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
```

Comment:
The libuv transport always closes the FileHandleEndPoint when it unbinds. If we're going to add this API, we should probably have libuv respect it.

A test to verify there's really a difference in behavior would also be nice.

Reply:
You want me to do extra work? OK

Reply:
If it's really too difficult, you could throw a NotSupportedException or something from libuv when OwnsHandle is false.

Reply:
I think that’s what I’ll do